### PR TITLE
docs: add global tags for openapi

### PR DESCRIPTION
**Pull request by pyphilia**
- add cli to generate html report from vacuum
- add global tags definition

### CODE REVIEW

File: src/plugins/swagger.ts [hunk 1]
```diff
@@ -13,6 +13,51 @@ export default async function (instance: FastifyInstance): Promise<void> {
...
+          name: 'password',
+          description: 'Endpoints related to authentication and management of password',
+        },
+        {
+          name: 'profile',
+          description: 'Endpoints related to user profile',
+        },
+        { name: 'recycled', description: 'Endpoints related to recycled items' },
+        { name: 'short-link', description: 'Endpoints related to short links to item' },
+        { name: 'tag', description: 'Endpoints related to visibility tags' },
```
Comment (spaenleh) [inline src/plugins/swagger.ts:58]: To change once the renaming PR is merged.

**Review by spaenleh — CHANGES_REQUESTED** [review]
Good work, I propose a few changes to the descriptions. 📖

**Review by ReidyT — APPROVED** [review]
Really nice!

File: package.json [hunk 2]
```diff
@@ -46,7 +46,8 @@
     "migration:revert:fake": "npx typeorm-ts-node-commonjs migration:revert --fake",
     "migration:check": "npx typeorm-ts-node-commonjs migration:generate src/migrations/migrations --check -d src/plugins/datasource.ts",
     "openapi:generate": "node dist/scripts/generateOpenAPI.js",
-    "openapi:lint": "vacuum lint ./openapi.json"
+    "openapi:lint": "vacuum lint ./openapi.json",
+    "openapi:report": "vacuum html-report ./openapi.json vacuum-report.html"
```
[the remark this answers is not in the document]
Comment (pyphilia) [reply package.json:50]: I actually tried but it was not really working. Actually I think the generation is not so accurate too, but let's see for next time.

**Pull request merged**